client/src/pages/login/login/login.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import './login.css';
+import { Link } from 'react-router-dom' //Links Library from React Router
+
+function Login() {
```

Comment:
Change to a class, because we will be working with states later on
```suggestion
class Welcome {
  render() {
    return <h1>Hello, {this.props.name}</h1>;
  }
}
```

Reply:
Ignore this wont work